plugins/modules/async_status.yml

```diff
@@ -0,0 +1,70 @@
+# Copyright (c) 2022 Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+# This is a stub to satisfy the GitHub doc action, see the builtin docs for better up to date info
```

Comment:
I added a `seealso:` but maybe we can discuss with the docs folks about this page automatically render as or redirect to, the builtin.

@felixfontein any thoughts?

Reply:
The seealso definitely helps. Having these entries should at least fix up these pages on the official docs

![image](https://user-images.githubusercontent.com/8462645/192171903-49d1f132-bbf2-4ae6-ba77-1c7352f163e6.png)

I'm fine with a redirect to keep things on the same page as it has been historically. Maybe having them separate might even be helpful when things start to differ from the Windows functionality and the POSIX variants.